workflow/update.go

```diff
@@ -16,11 +16,7 @@ import (
 	"github.com/ava-labs/apm/util"
 )
 
-var (
-	_ Workflow = &Update{}
-
-	// TODO configurable branches
```

Comment:
TODO is stale, configurable branches was added in [this](https://github.com/ava-labs/apm/commit/655edabcdd7fa1d2e9418dd16f3a375370fe1fb2) commit.